src/html/index.html

```diff
@@ -0,0 +1,71 @@
+{#
+  Nunjucks Templating!
+  Homepage: https://mozilla.github.io/nunjucks/
+  Docs: https://mozilla.github.io/nunjucks/api.html
+#}
+
+{% extends 'layouts/application.html' %}
+
+{% block content %}
+<div id="top-fold">
+  <div id="title-box">
+  <p id="subtitle">A Viget Exploration</p>
+    <h1 id="title">
+      <a href="/">
+        <img src="images/title.svg" alt="Improving Medical Adherence" />
+      </a>
+    </h1>
+    <img src="images/yellow-pill.svg" alt="Yellow pill." id="yellow-pill">
+    <img src="images/pills.svg" alt="Green and light yellow pills." id="pills">
+    <img src="images/more-pills.svg" alt="Three pill tablets." id="more-pills">
+    <img src="images/purple-pills.svg" alt="Two purple pills." id="purple-pills">
+  </div>
+    <nav class="pillbar">
+      <ul>
+        <a href="#box-one"><li class="left-end active">The roblem</li></a>
+        <a href="#key-factors" onclick="scroll(1)"><li>Key Factors</li></a>
+        <a href="#questions"><li>The Question</li></a>
+        <a href="#solution"><li class="right-end ">The Solution</li></a>
+      </ul>
+    </nav>
+
+    <div class="box" id="box-one">
+      <p id="main-thought">Patients who don’t take medication as prescribed are at risk of worsening health, increased hospitalization, and a greater chance of mortality.</p>
+      <p>Just in the U.S., non-adherence has been attributed to over $100 billion dollars in avoidable healthcare costs.<sup>1</sup> In developed countries, among patients with chronic illness, approximately 50% do not take medications as prescribed.<sup>2</sup></p>
+
+      <div class="quote">
+        <p id="quote-text">“Increasing the effectiveness of adherence interventions might have a greater impact on the health of the population than any improvement in specific medical treatment.”</p>
+        <p class="attribution">World Health Organization, 2005 <sup>3</sup><p>
+      </div>
+      <img src="images/pill-bottle.svg" alt="Short yellow pill bottle." id="pill-bottle">
+      <img src="images/pill-bottle2.svg" alt="Tall yellow pill bottle." id="pill-bottle2">
+      <img src="images/pill-bottle3.svg" alt="Red medicine bottle." id="pill-bottle3">
+    </div>
+
+    <div id="key-factors">
+      <h2 class="section-title">Why People Don't Adhere</h2>
+      <p class="section-intro">A myriad of factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</p>
+      <img src="images/body.svg" alt="Simple human body." id="body">
```

Comment:
Miiiiight be able to have this be a background image on the `#key-factors` div, since it's just visual decoration.